### ISSUE TITLE
New inventory system by MercyfullFate

### ISSUE THREAD

**Issue by JoSchaap** [JoSchaap/GoT_Wasteland_V2.Stratis#80]
MercyfullFate re-wrote the base code handling the player inventory and using the items inside of it.
It needs some proper testing/adjustment before we can implement this.

@MercyfulFate is it ok if i postpone these changes till 2.4 ?
stable branch got updated today so i want to follow in a release of GoT Wasteland v2.3 :)

**Comment by MercyfulFate** [comment]
If you want.
The errors you sent me make no sense... I need more information to troubleshoot them as I don't have any issues on either dev or stable branches.

**Comment by JoSchaap** [comment]
ok parked the work we got sofar here: https://github.com/JoSchaap/GoT_Wasteland_V2.Stratis/tree/%23backup_NewInventory_v24

the issue was the respawn dialog not popping up.
After releasing v2.3 ill get right back on this because i did like the changes you did :)

**Comment by MercyfulFate** [comment]
I looked into it... The issues you had look like it was because "mf_init" and "mf_compile" wasn't defined... yet it was defined about 20 lines about the error :P...

I've got a gutting/cooking system as well on the way, as well as the Warchest (allows players to share money with the team, and other teams to hack into it and steal some).

If you or anyone else has any ideas I'm happy try to implement them.

**Comment by JoSchaap** [comment]
the warchests i dont like but the gutting would be cool, there should be new animals too :)

**Comment by MercyfulFate** [comment]
any particular aspect of the warchest you dont like? I reason why I wanted to implement it was to encourage better  teamplay/base-building as atm most of the servers I play in just seem like no matter what side you are on, its just all lone wolves. If you have any ideas towards that, I'm all ears.

**Comment by MercyfulFate** [comment]
Also should note that I liked the idea of warchests because otherwise the only option is to drop all your cash, which is not normally recognized as the "Team's property", that and the drop down box (in the player menu) is just painful :P.

**Comment by AgentRev** [comment]
I like the warchest idea, looking forward to it.

**Comment by MercyfulFate** [comment]
The warchest code is almost complete. I used the standard ARMA3 GUI defines from the GUI Editor export, so the classes need to be converted to your wasteland GUI classes. The rest of the code is good to ship.

gutting should be completed today, and should be able to customize how much meat you get from each type of animal.

**Comment by JoSchaap** [comment]
just create a pull on the backup branch i made. i might just try the warchest before having an opinion about it :)

https://github.com/JoSchaap/GoT_Wasteland_V2.Stratis/tree/%23backup_NewInventory_v24

oh and wasteland just overrules the default gui classes so it will be instantly transformed to the green look :)

**Comment by MercyfulFate** [comment]
Arma 3 must have added a class that I use for the background.

Are you sure that the inventory was causing the issues? I can't reproduce the issues at all.

**Comment by JoSchaap** [comment]
as soon as i get v2.3 done ill jump right onto this :)

**Comment by s3kShUn61** [comment]
@MercyfulFate one idea branching off slightly from the warchest would be the ability for teams to buy base building supplies such as like a gunstore or general store. A 'fund' such as the warchest could be used to pay for the base building supplies.

**Comment by MercyfulFate** [comment]
I definitely thought about that, but then it removes the element of risk when going to purchase. pretty keen to hear any ideas/extensions on this, or any other ideas for that matter.

I am looking into a base supply shop atm, but im back at uni now and working full time so my spare time just vanished :P

**Comment by SCETheFuzz** [comment]
I know I been trying to set up a car shop that players can buy from rather then having a lot of cars spawn around the map.
